core/camel-core/src/main/java/org/apache/camel/reifier/ProcessorReifier.java

```diff
@@ -187,11 +187,15 @@
     protected final Logger log = LoggerFactory.getLogger(getClass());
 
     protected final T definition;
-    
+
     public ProcessorReifier(T definition) {
         this.definition = definition;
     }
 
+    public static void registerReifier(Class<?> dataFormatClass, Function<ProcessorDefinition<?>, ProcessorReifier<? extends ProcessorDefinition<?>>> creator) {
```

Comment:
This is a copy/paste mistake with data format ;)

Reply:
should be fixed now